pandas/core/arrays/datetimes.py

```diff
@@ -1119,7 +1119,7 @@ def date(self):
     weekday_name = _field_accessor(
         'weekday_name',
         'weekday_name',
-        "The name of day in a week (ex: Friday)\n\n.. deprecated:: 0.23.0")
+        "The name of day in a week (ex: Friday).. deprecated:: 0.23.0")
```

Comment:
Those are different from the previous, and are required, can you restore the original here please?

Reply:
Yes, sorry about that